pages/linux/grub2-mkpasswd-pbkdf2.md

```diff
@@ -0,0 +1,8 @@
+# grub2-mkpasswd-pbkdf2
+
+> You can set up a password to prevent unauthorized users from accessing the GRUB command line, modifying kernel command-line arguments, or booting non-default OSTree deployments.
```

Comment:
```suggestion
> Set up a password to prevent unauthorized users from accessing the GRUB command line, modifying kernel command-line arguments, or booting non-default OSTree deployments.
```

Reply:
changes done.